### PR TITLE
put zarf config file in the zarf config folder

### PR DESCRIPTION
**Pull request by corang**
I think there might be some weirdness as far as where we think we're executing zarf from vs where zarf thinks it's being executed from, so I put the config file in a location I know it'll check

### CODE REVIEW

**Comment by corang** [comment]
Changing it to use double quotes and a tee instead of bash redirection seems to have worked

**Review by RothAndrew — CHANGES_REQUESTED** [review]

File: test/e2e/utils/utils.go [hunk 1]
```diff
@@ -73,8 +73,12 @@ func SetupTestPlatform(t *testing.T, platform *types.TestPlatform) {
...
 		require.NoError(t, err, output)
 
-		// Add config file to ~/app/build
-		output, err = platform.RunSSHCommandAsSudo(`echo '"'"'no_progress = true'"'"' > ~/app/build/zarf-config.toml`)
+		// Create Zarf config folder
+		output, err = platform.RunSSHCommandAsSudo(`mkdir -p ~/.zarf`)
+		require.NoError(t, err, output)
+
+		// Add config file to Zarf config folder
+		output, err = platform.RunSSHCommandAsSudo(`echo "no_progress = true" | tee ~/.zarf/zarf-config.toml`)
```
[the remark this answers is not in the document]
Comment (RothAndrew) [reply test/e2e/utils/utils.go:81]: `output, err = platform.RunSSHCommandAsSudo(`mkdir -p ~/.zarf && echo "no_progress = true" > ~/.zarf/zarf-config.toml`)`

[the remark this answers is not in the document]
Comment (RothAndrew) [reply test/e2e/utils/utils.go:81]: Just saw your earlier comment about tee. If that's the way it needs to happen that's fine, it's a very common command to have installed on most/all linux OSes

[the remark this answers is not in the document]
Comment (corang) [reply test/e2e/utils/utils.go:81]: I couldn't get it to work using `>`, switching to tee got the file to actually be written. I'm not against combining them though.

[the remark this answers is not in the document]
Comment (RothAndrew) [reply test/e2e/utils/utils.go:81]: okay no sweat. Let's just get it as one command then and get it merged

**Pull request merged**